fenzo-core/src/main/java/com/netflix/fenzo/queues/tiered/QueueBucket.java

```diff
@@ -130,6 +138,14 @@ public double getDominantUsageShare() {
                 Math.max(TierSla.eps / 10.0, allocsShareGetter.apply(tierNumber, name));
     }
 
+    public ResAllocs getUsedOrGuaranteed() {
+        return ResAllocsUtil.ceilingOf(totalResAllocs, totals.asResAllocs());
+    }
+
+    public boolean isBelowGuaranteedConsumption() {
+        return ResAllocsUtil.isLess(totals.asResAllocs(), totalResAllocs);
```

Comment:
The totalResAllocs is the totals for the tier. Comparing with it isn't right to see if this bucket is below its guaranteed consumption. This bucket's share from the total is in TierSlas object.